.prettierignore

```diff
@@ -1,3 +1,2 @@
 node_modules
-src/**/*.mjs
-src/**/*.d.mts
+src/**/*.js
```

Comment:
We still want the *.d.ts as prettier will run on the compiled `*.d.ts` files in CI without this ignore directive

Reply:
Good catch, I missed that when resolving the merge